docs/conf.py

```diff
@@ -16,19 +16,17 @@
 import sys
 from datetime import datetime
 
-
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.insert(0, os.path.abspath('..'))
 
 import mechanicalsoup
 
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
```

Comment:
I disagree with this series of changes. I like the distinction between comments (`# like this`) and commented-out code (`#like = that`). No strong opinion, but I'd rather not change things I'm already happy with.